src/pages-and-resources/discussions/app-config-form/apps/legacy/LegacyConfigForm.test.jsx

```diff
@@ -192,4 +201,140 @@ describe('LegacyConfigForm', () => {
         expect(container.querySelector(`#checkbox-${id}`)).toBeChecked();
       });
     });
+
+  const createNewTopic = (topicName) => {
+    const currentTopics = defaultAppConfig.discussionTopics.length; // #2
```

Comment:
`totalTopics`?

Reply:
I feel like topicIndex will be more suitabe here

Reply:
agree with you here.